src/ImageView.jl

```diff
@@ -504,13 +504,16 @@ function prep_contrast(img::Signal, clim::Signal{CLim{T}}) where {T}
     # Return a signal corresponding to the scaled image
     imgc = map(img, clim; name="clim-mapped image") do image, cl
         cmin, cmax = safeminmax(cl.min, cl.max)
-        mtyp = T<:GrayLike ? Gray{N0f8} : T
-        smm = scaleminmax(mtyp, cmin, cmax)
+        smm = scaleminmax(outtype(T), cmin, cmax)
```

Comment:
This may not be strictly necessary, but it coerces images into a display format (`N0f8` storage) which catches some problems earlier and uses less memory than, e.g., `RGB{Float64}`.